fenix/app/src/main/res/xml/default_engine_preferences.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
**nit:** Let's rename this to be `default_search_engine_preferences` so that the name will map to what we see when viewing the fragment on the device.